packages/react-bootstrap-table2/test/header-cell.test.js

```diff
@@ -436,6 +436,233 @@ describe('HeaderCell', () => {
           });
         });
       });
+
+      describe('when headerSortingClasses was defined ', () => {
+        const classes = 'foo';
+        const order = Const.SORT_DESC;
+
+        describe('if headerSortingClasses is a string', () => {
+          beforeEach(() => {
+            column = { ...column, headerSortingClasses: classes };
+            wrapper = shallow(
+              <HeaderCell
+                column={ column }
+                index={ index }
+                sorting
+                sortOrder={ order }
+              />);
+          });
+
+          it('should append classes correcly', () => {
```

Comment:
should + V-ing

Reply:
I thought that will be great for should + VR :)